flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/ResultSubpartition.java

```diff
@@ -136,13 +136,13 @@ public void initializeState(ChannelStateReader stateReader) throws IOException,
 		private final Buffer buffer;
 		private final boolean isMoreAvailable;
 		private final int buffersInBacklog;
-		private final boolean nextBufferIsEvent;
+		private final boolean isAvailableWithoutCredit;
```

Comment:
it is ambiguous to distinguish the semantics between `isAvailableWithoutCredit` and `isMoreAvailable` if not back to see the implementation. I prefer to using `dataAvailable` and `eventAvailable` instead.